### PR TITLE
Update page in Connect AWS Gateway “deploy-your-agents-1”

### PR DESCRIPTION
**Pull request by dfeldick**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by dfeldick** [comment]
https://jira.axway.com/browse/APIGOV-16553

**Comment by dfeldick** [comment]
@sbolosan @jcollins-axway @kflan-io @vivekschauhan  please review

**Comment by lbadenhop** [comment]
@dfeldick Did you mean to unbold AMPIFY Central variables?

**Review by cbordier — APPROVED** [review]
I approve the change.
@lbadenhop, please proceed.

**Comment by lbadenhop** [comment]
Conflicts with other changes. A new PR will be created.

File: content/en/docs/central/connect-aws-gateway/deploy-your-agents-1.md [hunk 1]
```diff
@@ -85,6 +82,7 @@ The configuration of the AWS_QUEUENAME is not used in this mode.
...
 | CENTRAL_SSL_INSECURESKIPVERIFY                                     | InsecureSkipVerify controls whether a client verifies the server's certificate chain and host name. If true, TLS accepts any certificate presented by the server and any host name in that certificate. In this mode, TLS is susceptible to man-in-the-middle attacks.                                                |
+| CENTRAL_SUBSCRIPTIONS_ISSUENEWCREDENTIALS                          | When creating a subscription on AMPLIFY Central, setting this value to true will cause the agent to create new authorization credentials for the API Manager application. If false, new subscriptions will use a previously defined authentication credential from the application. The value defaults to true.                                                                                                                                                                                                          |
```
Comment (jcollins-axway) [inline content/en/docs/central/connect-aws-gateway/deploy-your-agents-1.md:85]: @dfeldick Is this supposed to be here still?

**Pull request closed without merging**